src/components/Sing/SequencerNote.vue

```diff
@@ -386,6 +396,10 @@ const onLyricInput = (event: Event) => {
   left: -1px;
   width: 5px;
   height: 100%;
+
+  &:hover {
+    background-color: #80d998;
+  }
```

Comment:
この形だと実際に長さを変更しているときにhoverになったりならなかったりで振動しちゃってそうでした！
例えばプレビュー中に明示的にクラス指定すると良いのかなと。

ということでFIXMEコメントだけつけさせていただきます！

```suggestion
  &:hover {
    // FIXME: hoverだとカーソル位置によって適用されないので、プレビュー中に明示的にクラス指定する
    background-color: #80d998;
  }
```

Reply:
@Hiroshiba 
(お気持ちのみ・やるのちょっと面倒そう)
こちらドラッグ中や他移動中などではできればスタイルとカーソルを固定したいと思っております！